_posts/2018-06-13-get-ready-to-rebble.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Getting Ready for Rebble"
+date:   2018-06-13 12:00:00
+# categories: community
+---
+
+Hi again! I'm [Katharine Berry](https://twitter.com/KatharineBerry), back
+with a brief update about the Rebble Web Services. As you have all probably noticed,
+we've been a bit quiet since my [last post](/2018/02/15/rebble-web-services.html).
+Unfortunately, life happened for all of us, as it tends to. But we're back on
+the case, and before we go any further, we have something important for you to
+do!
+
+## Rebble Accounts
+
+The Rebble account system is up and running, and now is the time for you to
+create your accounts. **Head over to [Rebble Auth](https://auth.rebble.io)
+and log in using your preferred site.** You'll then be asked to link your
+Pebble account. Please do: we will use this to import information from the
+Pebble servers before they shut down in order to help make the transition
+smoother.
+
+**It is especially important for developers to link their Pebble accounts now.**
+Once the Pebble services shut down at the end of this month, we will no longer
+be able to identify developers who did not link their accounts, and so we will
+be unable to let them update or otherwise change their apps.
+
+We are also going to use the number of accounts created to assist us in
+determining service usage and attempt to validate our assumptions about the
+number of users we expect to see.
+
+That's all there is to do right now: there is no app to install, and we are not
+yet ready for users to switch over to Rebble services. We will be sure to update
+you when we are!
+
+So to repeat myself: **please go and
+[make your Rebble account](https://auth.rebble.io) right now.** You must do so
+before the end of this month, and it only takes a minute!
+
+Okay, now that that's out of the way...
+
+## The State of Rebble Web Services
+
+So here's the bad news: we are probably not going to have _everything_ in place
```

Comment:
I understand the need to manage expectations, but I feel like we should strive for certainty when possible - perhaps enumerating services already online or underway, and mentioning the less likely along with at least a rough notion of their impact?